### PR TITLE
Feature/send gmail sms

### PR DESCRIPTION
**Pull request by buigiaanfb1**
## Screenshot/Record (FE)
Screen Shot 2023-04-02 at 02 32 54
Screen Shot 2023-04-02 at 02 33 02
Screen Shot 2023-04-02 at 02 33 33

## Checklist before requesting a review
1. [ ] I have performed a self-review of my code
2. [ ] Have you lint your code locally before submission?

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -418,6 +419,22 @@ model TagsNews {
   @@map("tags_news")
 }
 
+model Ping {
```
Comment (pvnamGit) [inline prisma/schema.prisma:422]: I believe it should be a m-n relationship. Let imagine, user A can send a message to n users, user A also can received n messages from n users.

File: src/modules/profiles/components/ProfileSidebar.tsx [hunk 2]
```diff
@@ -147,6 +159,22 @@ const ProfileSidebar = () => {
               </Link>
             );
           })}
+          {alreadySendMessage ||
```
Comment (pvnamGit) [inline src/modules/profiles/components/ProfileSidebar.tsx:162]: Should align left to consistent with "Thong Tin"

File: src/modules/profiles/service/ping.serivce.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+          },
+        },
+      },
+    });
+
+    console.log(userInformation);
+
+    if (userInformation?.phone) {
+      await sendSmsService({
+        to: `+84${userInformation?.phone}`,
```
Comment (pvnamGit) [inline src/modules/profiles/service/ping.serivce.ts:38]: I have no idea how it works. But if my phone number is `03344xxxx`, so the destination of the message will be `+8403344xxxx`, right? Will the message be sent to the right destination?

File: src/modules/profiles/types.ts [hunk 4]
```diff
@@ -84,6 +84,11 @@ export type CreateCareerServiceProps = {
   endDate: string;
 };
 
+//carrers
```
Comment (pvnamGit) [inline src/modules/profiles/types.ts:87]: move this comment to line 91 bro :v

File: src/modules/profiles/service/ping.serivce.ts (hunk 3 above)
Comment (buigiaanfb1) [reply src/modules/profiles/service/ping.serivce.ts:38]: yeah i was concerned about that too. But ye, it works fine! No worries

**Pull request merged**